src/stories/common/ButtonWithIcon.stories.js

```diff
@@ -23,16 +23,34 @@ const Template = (args) => ({
         return { faStar, faFlag, faPlus, args };
     },
     template:
-        `<button-with-icon v-bind="args">
-            <template #text>Test Button</template>
-            <template #icon>
-                <fa class="text-blue-400 mr-3" size="2x" :icon="faFlag"/>
-            </template>
-        </button-with-icon>`,
+        '<button-with-icon v-bind="args" :icon="faStar">Text</button-with-icon>'
```

Comment:
```suggestion
        `<button-with-icon v-bind="args">
            <template #text>Test Button</template>
            <template #icon>
                <fa class="text-blue-400 mr-3" size="lg" :icon="faFlag"/>
            </template>
        </button-with-icon>`,
```

This will by default set the icon size to lg. However it will scale up if `button-with-icon` the container has a larger font size set.